src/Client/Web/Bit.Client.Web.BlazorUI/Components/Checkboxes/BitCheckbox.razor.cs

```diff
@@ -108,5 +110,38 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
 
             await base.OnAfterRenderAsync(firstRender);
         }
+
+        public override Task SetParametersAsync(ParameterView parameters)
+        {
+            foreach (ParameterValue parameter in parameters)
+            {
+                switch (parameter.Name)
```

Comment:
The private field holding the state of value of 2-way bound parameter should be reset at the start of this method

Reply:
Done